lib/controller.js

```diff
@@ -170,6 +177,134 @@ Tessel.get = function(opts) {
     }
   });
 };
+/*
+Because of sync problem with master trunk and parallel development on Tessel.list 
+it was necessary to go my own way...
```

Comment:
@Student007 do you need some assistance rebasing this commit on master? How can we take advantage of the work that was done on master while you were working on this? What of this new code can we remove?